filters/all.js

```diff
@@ -98,7 +99,7 @@ function isDefined(obj) {
 filter.isDefined = isDefined;
 
 function isProtocol(api, protocol){
```

Comment:
can you remind me what is this filter responsible for?

Reply:
@derberg check if arg `protocol` is mentioned in async-api file as [expected protocol](https://www.asyncapi.com/docs/specifications/v2.0.0#definitionsProtocol). Depend on which protocol user defined in async-api different configurations and dependencies will be generated.